File: herencia-dto/HerenciaDto-Java-IvanRC/src/main/java/models/Alumno.java
package models;

public final class Alumno extends Persona {
    static Integer edad;

    public Alumno(String nombre, Integer edad) {
        super(nombre);
        this.edad = edad;
    }

    public static Integer getEdad() {
        return edad;
    }

    public void setEdad(Integer edad) {
        this.edad = edad;
    }

    @Override
    public String toString() {
        return "Alumno{" +
                "edad=" + edad +
                ", nombre='" + nombre + '\'' +
                '}';
    }

    public static <T> int getEdad(T t) {
        return getEdad();
    }
}

File: herencia-dto/Herencia_OscarGrC_Kotlin/src/main/kotlin/models/Alumno.kt
package models

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root
import java.io.Serializable

data class Alumno(
    var edad:Int,
    override var nombre:String): Persona(), Serializable {
    override fun toString(): String {
        return "Hola soy ${nombre}, soy Alumno de ${edad} años \n"
    }
}